packages/multichain-account-service/src/types.ts

```diff
@@ -122,3 +138,31 @@ export type MultichainAccountServiceMessenger = RestrictedMessenger<
   AllowedActions['type'],
   AllowedEvents['type']
 >;
+
+export type Bip44AccountProvider = AccountProvider<
+  Bip44Account<KeyringAccount>
+>;
+
+/**
+ * Locally widen the withKeyring action so the messenger accepts the optional options
+ * parameter (createIfMissing/createWithData) in addition to selector and operation.
+ */
+export type KeyringControllerWithKeyringWithOptionsAction = {
```

Comment:
This relaxed action type was created to circumvent the function signature that typescript defaults to which is without options.